Cargo.toml

```diff
@@ -9,9 +9,25 @@ authors = ["The ocicrypt Authors"]
 edition = "2018"
 
 [features]
-default = ["keywrap-jwe"]
+default = ["keywrap-jwe","keywrap-keyprovider","utils-runner","utils-keyprovider"]
 keywrap-jwe = []
+keywrap-keyprovider = []
+utils-runner = []
+utils-keyprovider = []
 
 [dependencies]
 anyhow = ">=1.0"
 josekit = ">=0.7"
+serde = "1.0"
+serde_derive = "1.0"
+serde_json = "1.0"
+bincode = "1.2.1"
+tonic = "0.5.0"
+prost = "0.8.0"
+prost-types = "0.7.0"
+aes-gcm = "0.9.2"
+base64 = "0.13.0"
+tokio = { version = "1.9.0", features = ["full"]}
+
+[build-dependencies]
+tonic-build = "0.5.1"
```

Comment:
Sure I will be adding ttrpc implementation in my next PR. I initially started with grpcio, but I saw few blogs saying tonic is more reliable and fast production ready. https://blog.logrocket.com/rust-and-grpc-a-complete-guide/
I am ok to go for grpcio let me know your opinion

Reply:
IMO library should have protobufs and structs, agents should use the structs and grpc methods defined in library, otherwise it would lead to inconsistency in future.

Reply:
> Sure I will be adding ttrpc implementation in my next PR. I initially started with grpcio, but I saw few blogs saying tonic is more reliable and fast production ready. https://blog.logrocket.com/rust-and-grpc-a-complete-guide/
> I am ok to go for grpcio let me know your opinion

tonic is ok for current keyprovider with GRPC support, ttrpc will be our option for performance and footprint.